x-pack/test/functional/apps/ml/short_tests/notifications/notification_list.ts

```diff
@@ -92,5 +96,22 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await browser.refresh();
       await ml.notifications.assertNotificationErrorsCount(0);
     });
+
+    it('support custom sorting for notifications level', async () => {
```

Comment:
nit: `support` → `supports`

Reply:
Fixed in https://github.com/elastic/kibana/pull/153462/commits/757f797ada55aaaf0e820289c9b68bcc64fd5fb2